### DISCUSSION TITLE
Server connections PC vs. Mac

### DISCUSSION THREAD

**Discussion by lorbeerbund** [jamulussoftware/jamulus#1531] [General]
Good evening,

I am new to Jamulus with the aim of offering my choir a way to make music online. So far I have set up two clients, one on my PC (Linux Mint via LAN) and one on a slightly older Macbook (via WLAN). Everything works fine and I can connect to servers.

The first thing I notice is that many more servers can be reached from the Macbook than from the Linux machine. About twice as many servers can be reached from the Macbook than from the PC. Can someone explain to me why this is happening?

Many greetings
lorbeerbund

**Comment by dcorson-ticino-com** [comment]
Hi Lorbeer,
The way this works is the following:
Your PC, the client, requests from the selected directory server the list of servers that the directory server manages.
When it receives that list your PC then sends a ping to all the servers on the list.
The pinged server is shown in the Connect dialog when your client has received a ping response.
If one of your PCs is not seeing servers, then that PC has network connection problems.  Either the pings are not transmitted correctly or the ping responses are not arriving at your PC.
This does not depend on the PC type, it is the same for Win10, MAC and Linux.
This is also congruent with the sound problems you mentioned elsewhere.  Jamulus can not correct for missing data.

**Comment by lorbeerbund** [comment]
Thanks a lot, @dcorson-ticino-com!

**Reply by wolli** [reply]
Gruß vom Zwitscher Vögelchen an Lorbeerbund.
Bin auch neu dabei und übe noch.

LG
